test/593-early-path.py

```diff
@@ -8,4 +8,4 @@
 for z, x, y in tiles:
     assert_has_feature(
         z, x, y, 'roads',
-        {'highway': 'path', 'name': true})
\ No newline at end of file
+        {'highway': 'path', 'name': None})
```

Comment:
Yeah, that's slightly confusing, but having `None` in the properties means that [the key must be present](https://github.com/mapzen/vector-datasource/blob/master/test.py#L52). Having `true` here was an "unknown variable" error (Python calls it `True` with a capital T), but `True` would have checked for a literal (JSON) `true`, not for any value.